storage-s3-dynamodb/integration_tests/dynamodb_logstore.py

```diff
@@ -45,7 +45,7 @@
 export DELTA_NUM_ROWS=16
 
 ./run-integration-tests.py --run-storage-s3-dynamodb-integration-tests \
-    --dbb-packages org.apache.hadoop:hadoop-aws:3.3.1,com.amazonaws:aws-java-sdk-bundle:1.12.142 \
+    --dbb-packages org.apache.hadoop:hadoop-aws:3.3.1 \
```

Comment:
`com.amazonaws:aws-java-sdk-bundle:1.12.142` is a compile dependency for `org.apache.hadoop:hadoop-aws:3.3.1`